conduit/src/Data/Conduit/Combinators.hs

```diff
@@ -1755,6 +1756,22 @@ mapAccumWhileC f =
 {-# INLINE mapAccumWhileC #-}
 STREAMING(mapAccumWhile, mapAccumWhileC, mapAccumWhileS, f s)
 
+
+-- | Specialized version of 'mapAccumWhile' that does not provide values downstream.
+--
+-- Subject to fusion
+--
+-- @since TODO
```

Comment:
Can you add a minor version bump, changelog entry, and fill in this `@since` comment?